ruby/vampires.rb

```diff
@@ -0,0 +1,125 @@
+# Match the following conditions in the order they're listed. Your program should base its result on the latest condition matched, not the first condition matched. In other words, it's not a "only one condition will apply" sort of scenario. This may mean repeatedly updating a variable as each condition is checked, so that the variable always contains the most precise result.
+
+# If the employee got their age right, and is willing to eat garlic bread or sign up for insurance, the result is “Probably not a vampire.”
+# If the employee got their age wrong, and hates garlic bread or waives insurance, the result is “Probably a vampire.”
+# If the employee got their age wrong, hates garlic bread, and doesn’t want insurance, the result is “Almost certainly a vampire.”
+# Even if the employee is an amazing liar otherwise, anyone going by the name of “Drake Cula” or “Tu Fang” is clearly a vampire, because come on. In that case, you should print “Definitely a vampire.”
+# Otherwise, print “Results inconclusive.”
+# Print the result at the end of the survey.
+
+def interview_vampire(i)
+
+	evaluated_employee = ""
+	employee_allergies = ""
+
+	puts "What is your name?"
+	interview_name = gets.chomp
+	puts "How old are you?"
+	interview_age = gets.chomp.to_i
+	puts "Our company cafeteria serves garlic bread. Should we order some for you? (yes/no)"
+	interview_garlic = gets.chomp.downcase
+	puts "Would you like to enroll in the company’s health insurance? (yes/no)"
+	interview_health = gets.chomp.downcase
+	
+
+	if interview_age < 100 && (interview_garlic == "yes" || interview_health == "yes")
+	
+		evaluated_employee = "probably not a vampire"
+	
+	elsif interview_age >= 100 && (interview_garlic != "yes" || interview_health != "yes")
+	
+		evaluated_employee = "probably a vampire"
+	
+	end 
+
+	if interview_age >= 100 && (interview_garlic != "yes" && interview_health != "yes")
+	
+		evaluated_employee = "almost certainly a vampire"
+	
+	end
+
+	if interview_name == "drake cula" || interview_name == "tu fang"
+		
+		evaluated_employee = "definitely a vampire"
+
+	end
+
+	if evaluated_employee == ""
+		
+		evaluated_employee = "Results inconclusive!"
+	
+	end
+
+	while true
```

Comment:
`while true` is kind of a cheat to get the looping structure to run. What you are really checking to see is if employee_allergies == 'done' || employee_allergies == 'sunshine', then you do not have to insert break statements into the code. The only trick is that employee_allergies has to be defined before you start looping.